github-runner-manager/src/github_runner_manager/manager/runner_manager.py

```diff
@@ -173,31 +185,18 @@ def get_runners(self) -> tuple[RunnerInstance, ...]:
             Information on the runners.
         """
         logger.debug("runner_manager::get_runners")
-        runner_instances = []
         cloud_runners = self._cloud.get_runners()
         runners_health_response = self._platform.get_runners_health(cloud_runners)
         logger.info("clouds runners response %s", cloud_runners)
         logger.info("runner health response %s", runners_health_response)
         runners_health = runners_health_response.requested_runners
         health_runners_map = {runner.identity.instance_id: runner for runner in runners_health}
-        for cloud_runner in cloud_runners:
-            if cloud_runner.instance_id not in health_runners_map:
-                runner_instance = RunnerInstance.from_cloud_and_platform_health(cloud_runner, None)
-                runner_instance.health = HealthState.UNKNOWN
-                runner_instances.append(runner_instance)
-                continue
-            health_runner = health_runners_map[cloud_runner.instance_id]
-            if health_runner.deletable:
-                cloud_runner.health = HealthState.UNHEALTHY
-            elif health_runner.online:
-                cloud_runner.health = HealthState.HEALTHY
-            else:
-                cloud_runner.health = HealthState.UNHEALTHY
-            runner_instance = RunnerInstance.from_cloud_and_platform_health(
-                cloud_runner, health_runner
+        return tuple(
+            RunnerInstance.from_cloud_and_platform_health(
+                cloud_runner, health_runners_map.get(cloud_runner.instance_id, None)
```

Comment:
```suggestion
            RunnerInstance.from_cloud_and_platform_health(
                cloud_instance=cloud_runner,
                platform_health_state=health_runners_map.get(cloud_runner.instance_id, None)
```

nit: kwargs would make it a bit more easier to follow